pandas/io/formats/latex.py

```diff
@@ -199,7 +199,7 @@ def _format_multicolumn(self, row: List[str], ilevels: int) -> List[str]:
         will become
         \multicolumn{3}{l}{a} & b & \multicolumn{2}{l}{c}
         """
-        row2 = list(row[:ilevels])
+        row2 = list(row[: self._ilevels])
```

Comment:
why is this not a list already?

Reply:
Indeedl. But I just re-used the original code as it was, with minimum intrusion.

Reply:
Fixed that.